caluma/faker.py

```diff
@@ -0,0 +1,17 @@
+from faker import Faker
```

Comment:
I would rename this file. Depending on where you import it from, it may not be clear if the actual `faker` package is imported or our own.

Reply:
faker is a submodule of caluma so I would need to import as `from caluma.faker import ...`

same thing happens with models.py, serializers.py etc. which also have the same naming as upstream so I would prefer keeping the naming to keep it consistent with upstream.